api/voice-react-native-sdk.api.md

```diff
@@ -874,6 +874,7 @@ export interface Voice {
 // @public
 export class Voice extends EventEmitter {
     constructor();
+    configureCallKit(configuration: Record<string, any>): Promise<void>;

```

Comment:
- Do we need to configure callkit once for all calls, or once for every call? Just wondering if this should go in the connect method.
- Is it possible to define the `Record`'s possible values instead of just `string, any`? Or are the values dynamic and could change in the future without needing to update the sdk?

Reply:
@charliesantos 
- The app can definitely make configuration changes from time to time but the options look more like a one-time setup.
- re: Record - since all the options are optional and have different types, `<string, any>` sounds more suitable for the case, unless there is way to force data type for such case?

Reply:
- In that case, +1 on making this it's own method. What are your thoughts on renaming this to `setCallKitConfiguration`?
- We can force data types, but whether it makes sense depends on the possible keys and values. What are the possible keys and values for each key? Will these keys and values change dynamically in the future?

Reply:
Yes, options can change in the future when Apple finds it necessary. I added supported options in the `@remarks` of the method in `Voice.tsx` but would like you and @mhuynh5757 to help structure the docs so it makes sense for the RN developers.